content/en/docs/concepts/storage/storage-classes.md

```diff
@@ -121,16 +121,23 @@ allows the users to resize the volume by editing the corresponding PVC object.
 The following types of volumes support volume expansion, when the underlying
 Storage Class has the field `allowVolumeExpansion` set to true.
 
-* gcePersistentDisk
-* awsElasticBlockStore
-* Cinder
-* glusterfs
-* rbd
-* Azure File
-* Azure Disk
-* Portworx
-* FlexVolumes
-* CSI  {{< feature-state for_k8s_version="v1.14" state="alpha" >}}
+{{< table caption = "Feature state support for various storage types"  >}}
+
+Volume type | Feature state support since
+:---------- | :--------------------------
+gcePersistentDisk | 1.11
+awsElasticBlockStore | 1.11
+Cinder | 1.11
+glusterfs | 1.11
+rbd | 1.11
+Azure File | 1.11
+Azure Disk | 1.11
+Portworx | 1.11
+FlexVolumes | 1.13
+CSI | 1.14 ,1.15-alpha ,1.16-beta
```

Comment:
I don't know if a glossary tooltip would work here, but if it does then I'd use one (referencing [CSI](https://kubernetes.io/docs/reference/glossary/?storage=true#term-csi) in the glossary).

Reply:
When I added glossary tooltip, it pulled the CSI row out of the table. Formatting after that was off. If you are interested take a look at this preview - https://deploy-preview-16237--kubernetes-io-master-staging.netlify.com/docs/concepts/storage/storage-classes/

I reverted it back to original.